scalding-hadoop-test/src/test/scala/com/twitter/scalding/platform/PlatformTest.scala

```diff
@@ -342,15 +342,16 @@ class PlatformTest extends WordSpec with Matchers with HadoopSharedPlatformTest
       HadoopPlatformJobTest(new TypedPipeJoinWithDescriptionJob(_), cluster)
         .inspectCompletedFlow { flow =>
           val steps = flow.getFlowSteps.asScala
-          steps should have size 1
+          steps should have size 2
```

Comment:
actually, I expect 1 step still. This is a good unit test because we should not be adding an extra step in the hashjoin here, as far as I can tell.